src/queue/importQueue.ts

```diff
@@ -17,4 +17,4 @@ importQueue.on('error', err => {
   console.log('queue error: ', err);
 });
 
-module.exports = importQueue;
+export = importQueue;
```

Comment:
Not really import for this PR, but would it work to instead change `const importQueue` to `export const importQueue`

Reply:
Is there an easy way to do this that doesn't require a bunch of other changes? This was the closest thing I found for crossing over to ts without having to do a bit of an overhaul.